lib/email.js

```diff
@@ -34,16 +45,16 @@ exports.sendConfirmationEmail = function(host, email, confirmation, userid, cb)
 exports.contactSeller = function(host, from, to, message, listingId, cb) {
   // setup e-mail data with unicode symbols
   var mailOptions = {
-      from: '"noreply" <noreply@tuskmarketplace.com>', // sender address
+      from: '"pleasereply" <pleasereply@tuskmarketplace.com>', // sender address
```

Comment:
Yeah we already have one.
> On Sep 10, 2016, at 1:23 PM, Brian Cefali <notifications@github.com> wrote:
> 
> In lib/email.js <https://github.com/TuftsTusk/ringmaster/pull/18#discussion_r78280677>:
> 
> > @@ -34,16 +45,16 @@ exports.sendConfirmationEmail = function(host, email, confirmation, userid, cb)
> >  exports.contactSeller = function(host, from, to, message, listingId, cb) {
> >    // setup e-mail data with unicode symbols
> >    var mailOptions = {
> > -      from: '"noreply" <noreply@tuskmarketplace.com>', // sender address
> > +      from: '"pleasereply" <pleasereply@tuskmarketplace.com>', // sender address
> Are we making a pleasereply@tmp.com <mailto:pleasereply@tmp.com> email account?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TuftsTusk/ringmaster/pull/18/files/a10bf340a47edfc004aa498629e4c1855a685aba#r78280677>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACtOnSwsNXdFb1WNLg0E_cptdavhpf09ks5qoueEgaJpZM4J4TmX>.
>